data/polkit/io.snapcraft.snapd.policy

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE policyconfig PUBLIC
+ "-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/PolicyKit/1.0/policyconfig.dtd">
+<policyconfig>
+
+  <vendor>Snapcraft</vendor>
```

Comment:
I just adapted the policy file from snapd-glib, which had the same vendor value.  I'm not even sure if this ever gets displayed to the user.